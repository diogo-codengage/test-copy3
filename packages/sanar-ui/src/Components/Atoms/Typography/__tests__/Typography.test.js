
import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESTypography from '../'

it('renders correctly', () => {
    const component = (
        <ESTypography>
            <h1>ESTypography</h1>
        </ESTypography>
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
