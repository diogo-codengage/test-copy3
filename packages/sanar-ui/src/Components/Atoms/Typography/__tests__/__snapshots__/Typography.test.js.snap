// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  aria-label={null}
  className="ant-typography es-typography"
  style={
    Object {
      "WebkitLineClamp": null,
    }
  }
>
  ESTypography
</div>
`;
